modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -2,16 +2,12 @@
 // * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
 // * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
 [id='virt-importing-vm-prerequisites_{context}']
-= Prerequisites for importing a virtual machine into {VirtProductName}
+= Prerequisites for importing a virtual machine
 
 ifdef::virt-importing-rhv-vm[]
-Importing a Red Hat Virtualization virtual machine into {VirtProductName} has the following prerequisites:
+Importing a virtual machine from Red Hat Virtualization (RHV) into {VirtProductName} has the following prerequisites:
```

Comment:
I think this is too repetitive (with the second bullet point being "You must ensure that the RHV VM meets the following prerequisites"). I'm also not convinced that this needs to be its own module instead of a prerequisites section in the assembly itself.

Reply:
At the time I wasn't sure whether the VMware and RHV imports would have overlapping prerequisites. Now it seems that they can be maintained separately, so I will move them to the module.